components/molecules/Card.js

```diff
@@ -23,9 +23,20 @@ export default function Card(props) {
         return `${readingTime} min ${props.t["read"]}`;
     }
 
+    function selectCheck(obj){
+        let selection = window.getSelection();
+        if(selection.toString().length > 0){
+            obj.preventDefault();
+            return false;
+        }else{
+            return true;
+        }
+    }
+
     return (
-        <Link href={`/${props.t.getLocale}/[slug]`} as={`/${props.t.getLocale}/${props.post.slug}`}>
-            <div className={styles.card}>
+      <Link href={`/${props.t.getLocale}/[slug]`} as={`/${props.t.getLocale}/${props.post.slug}`}>
```

Comment:
Same ici, juste remettre les espaces pour que ce soit au même niveau 
```suggestion
        <Link href={`/${props.t.getLocale}/[slug]`} as={`/${props.t.getLocale}/${props.post.slug}`}>
```